scipy/interpolate/_rgi_cython.pyx

```diff
@@ -0,0 +1,62 @@
+import numpy as np
```

Comment:
Side-note: FYI, it is possible to factor those directives at the sub-module level or at the global module level.

I do not know if it make sense for the Cython implementations of Scipy but this is what we have done for scikit-learn in https://github.com/scikit-learn/scikit-learn/pull/21512.

Do you think it would help?

Reply:
An environment variable was added then to be able to activate some checks during development, see: https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/_build_utils/__init__.py#L69-L88.